testsuite/integration-arquillian/pom.xml

```diff
@@ -320,6 +320,157 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>mysql</id>
+            <properties>
+                <keycloak.connectionsJpa.database>keycloak</keycloak.connectionsJpa.database>
+                <keycloak.connectionsJpa.user>keycloak</keycloak.connectionsJpa.user>
+                <keycloak.connectionsJpa.password>keycloak</keycloak.connectionsJpa.password>
+                <keycloak.connectionsJpa.url>jdbc:mysql://${auth.server.db.host}/${keycloak.connectionsJpa.database}</keycloak.connectionsJpa.url>
+                <jdbc.mvn.groupId>mysql</jdbc.mvn.groupId>
+                <jdbc.mvn.artifactId>mysql-connector-java</jdbc.mvn.artifactId>
+                <jdbc.mvn.version>5.1.29</jdbc.mvn.version>
+                <docker.database.image>mysql:5.7.25</docker.database.image>
+            </properties>
+            <build>
+                <pluginManagement>
```

Comment:
The plugins should be part of the generic flow, just plugin execution would be conditioned on existence of `docker.database.image`. The only issue is that the various containers need their specific env variables, e.g.  MYSQL_DATABASE, POSTGRES_DB - and all of those can be specified for `start-container` execution (rather renamed to `start-db-container` to allow other container types like LDAP to be handled similarly).

The profiles `db-postgres`, `db-mysql` would only define the `docker.database.image` and `jdbc.mvn.*` properties.